doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -271,27 +271,27 @@ the possible following elements delimited by single quotes:
 
 - tokens of the form ``'/ '`` are translated into breaking point, in
   case a line break occurs, an indentation of the number of spaces after
-  the “ ``/``” is applied (2 spaces in the given example)
+  the “``/``” is applied (no indentation in the given example)
```

Comment:
This subsection is baffles me a bit.  Where is the example?  Is it the `If If true` above?  I don't see `[v` in it.  Would be great if you can make this clearer.

Reply:
> Is it the If If true above?

Yeah, I think so.

```
If If True
   |  then False 
   ^--^ ← 3 spaces by [v
```